view/frontend/layout/default.xml

```diff
@@ -2,11 +2,11 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceBlock name="head.additional">
-            <block class="JustBetter\Sentry\Block\SentryScript" before="-" template="JustBetter_Sentry::script/sentry.phtml" name="sentry.head.additional" />
+            <block class="JustBetter\Sentry\Block\SentryScript" before="-" template="JustBetter_Sentry::script/sentry.phtml" name="sentry.head.additional" cacheable="false"  />
```

Comment:
Using cacheable="false" disables Full page caching for the whole page
https://magento.stackexchange.com/a/103319

Maybe load the customer module with requirejs to get the customer info
https://magento.stackexchange.com/questions/134466/how-to-check-that-customer-is-logged-in-on-frontend-in-magento-2-1

Reply:
See commit https://github.com/justbetter/magento2-sentry/pull/55/commits/31cf934454b2d6ea6c20706eee8a6a71e750c6eb